ephios/core/templates/core/fragments/event_list_list_mode.html

```diff
@@ -93,14 +98,11 @@ <h5 class="mb-0 text-break">
                                     {% translate "to" %}
                                     {{ event.end_time|date:"SHORT_DATE_FORMAT" }}
                                 {% endif %}
-                            </div>
+                            </a>
                             <div class="grid-action d-none d-lg-flex flex-column justify-content-center">
```

Comment:
I think this div is unneccessary now? I also feel like there is too much whitespace between the signup count and the right border

Reply:
Yeah I do wanna keep it in case we add something there.